pkg/clusterconfig/clusterconfig.go

```diff
@@ -225,6 +225,22 @@ func NewAlbIngressControllerServiceAccount(clusterName, region, policyArn, permi
 	})
 }
 
+// NewAWSLoadBalancerControllerServiceAccount returns an initialised configuration
+// for creating an aws-load-balancer-controller service account
+func NewAWSLoadBalancerControllerServiceAccount(clusterName, region, policyArn, permissionsBoundaryArn string) (*v1alpha5.ClusterConfig, error) {
```

Comment:
Name is misleading. Should be something like `NewAWSLoadBalancerControllerServiceAccountConfig`

Reply:
Given the name of the package, I am not sure this is required? I even wonder if the linter might not give us a warning if we try to do that🤔


From: https://blog.golang.org/package-names
> Avoid stutter. Since client code uses the package name as a prefix when referring to the package contents, the names for those contents need not repeat the package name. The HTTP server provided by the http package is called Server, not HTTPServer. Client code refers to this type as http.Server, so there is no ambiguity.

Reply:
Might be. The function name makes me think it actually creates a service account. 

In any case, minor thing